src/services/structures/controllers/closeBannerDemandeConseillerAvisPrefet.ts

```diff
@@ -0,0 +1,63 @@
+import { Application } from '@feathersjs/express';
+import { Response } from 'express';
+import { ObjectId } from 'mongodb';
+import { IRequest } from '../../../ts/interfaces/global.interfaces';
+import { action } from '../../../helpers/accessControl/accessList';
+import service from '../../../helpers/services';
+
+const closeBannerDemandeConseillerAvisPrefet =
+  (app: Application) => async (req: IRequest, res: Response) => {
+    const idStructure = req.params.id;
+    try {
+      if (!ObjectId.isValid(idStructure)) {
+        res.status(400).json({ message: 'Id incorrect' });
+        return;
+      }
+      const structure = await app
+        .service(service.structures)
+        .Model.accessibleBy(req.ability, action.update)
+        .updateOne(
+          {
+            _id: new ObjectId(idStructure),
+            coordinateurCandidature: false,
+            'prefet.banniereValidationAvisPrefet': true,
```

Comment:
humm si plusieurs objects in array pas de soucis ?

Reply:
non il y a qu'un seul objet qui peut avoir banniereValidationAvisPrefet à true